xarray/tests/test_backends.py

```diff
@@ -1767,6 +1767,71 @@ def test_setncattr_string(self) -> None:
                 assert_array_equal(one_element_list_of_strings, totest.attrs["bar"])
                 assert one_string == totest.attrs["baz"]
 
+    def test_compression_encoding(self) -> None:
+        data = create_test_data(dim_sizes=(20, 40, 10))
+        compression_vals = (
```

Comment:
Maybe using `pytest.mark.parametrize` for the compression_vals here, otherwise it shows only on first failure.